extensions/dacpac/src/wizard/pages/dacFxSummaryPage.ts

```diff
@@ -134,4 +134,3 @@ export class DacFxSummaryPage extends BasePage {
 		});
 	}
 }
-
```

Comment:
removed by mistake? IIRC, our new rule is append an empty line at the end of file.